src/controllers/presenter.controller.ts

```diff
@@ -1,44 +1,12 @@
 import { NextFunction, Request, Response } from "express";
-import { Session } from "@companieshouse/node-session-handler";
 
 import * as config from "../config";
-import { ApplicationData } from "../model";
-import { PresenterKey, PresenterKeys } from "../model/presenter.model";
-import { getApplicationData, setApplicationData, prepareData } from "../utils/application.data";
-import { logger } from "../utils/logger";
-import { saveAndContinue } from "../utils/save.and.continue";
+import { getPresenterPage, postPresenterPage } from "../utils/presenter";
```

Comment:
I don't think we'll be able to share the controllers. We're currently working on a spike / POC to remove the session use from the Registration controllers for our next piece of work, so the controllers will diverge soon.
I'll tag @mwestacott and @mikemoore24 for thought/opinions.

Reply:
I guess when we get to that point we can re-assess and see how much the controllers differ. My immediate thoughts were 'we'll need to change our registration controllers soon for the session work' but I'm happy to see what the consensus is from Martin and Mike.

Reply:
ok happy to go with this, i'll mark as resolved